test/e2e/cypress/pageObject/databases-overview-po.js

```diff
@@ -0,0 +1,202 @@
+export * from './base-po.js';
+import * as basePage from './base-po.js';
+
+// Test Data
+export const hdqDatabase = {
+  sid: 'HDQ',
+  instances: [
+    {
+      name: 'vmhdbqas01',
+      id: '99cf8a3a-48d6-57a4-b302-6e4482227ab6',
+      state: '',
+    },
+    {
+      name: 'vmhdbqas02',
+      id: 'e0c182db-32ff-55c6-a9eb-2b82dd21bc8b',
+      state: 'ACTIVE',
+    },
+  ],
+};
+
+const nwqSystem = {
+  sid: 'NWQ',
+  ascsInstance: {
+    id: '25677e37-fd33-5005-896c-9275b1284534',
+  },
+};
+
+const hddDatabase = {
+  sid: 'HDD',
+  instance: {
+    instanceNumber: '10',
+    row: 1,
+  },
+};
+
+// Selectors
+
+const hdqDatabaseCell = `tr:contains("${hdqDatabase.sid}")`;
+
+const hddDatabaseCell = `tr:contains("${hddDatabase.sid}")`;
+
+const databaseInstace1 = `a:contains("${hdqDatabase.instances[0].name}")`;
+const databaseInstace2 = `a:contains("${hdqDatabase.instances[1].name}")`;
+
+const deletedSapSystemToaster = `p:contains("The SAP System ${nwqSystem.sid} has been deregistered.")`;
+
+const tableGroupRows = '.table-row-group > div.table-row';
+
+const cleanUpButtonModal =
+  '#headlessui-portal-root button:contains("Clean up")';
+
+const cleanUpButtons = 'button:contains("Clean up")';
+
+const activePill = 'span:contains("ACTIVE")';
+
+const getCleanUpButtonByIdAndInstanceIndex = (id, index) =>
+  `tbody tr:contains("${id}") + tr div[class="table-row-group"] div[class*="table-row border-b"]:nth-child(${
+    index + 1
+  }) span:contains("Clean up")`;
+
+export const visit = () => basePage.visit('/databases');
+
+// UI Validations
+
+export const hdqDatabaseIsNotDisplayed = () =>
+  cy.get(hdqDatabaseCell).should('not.exist');
+
+export const hdqDatabaseIsDisplayed = () =>
+  cy.get(hdqDatabaseCell).should('be.visible');
+
+export const hddDatabaseIsNotDisplayed = () =>
+  cy.get(hddDatabaseCell).should('not.exist');
+
+export const bothDatabaseInstancesAreDisplayed = () => {
+  cy.get(databaseInstace1).should('be.visible');
+  return cy.get(databaseInstace2).should('be.visible');
+};
+
+export const activePillIsDisplayedInTheRightHost = () => {
+  return hdqDatabase.instances.forEach((instance) => {
```

Comment:
The multiple return statements look a bit wild to me. 
Maybe we could use cy.wrap ? and each instead of the forEach? 
```  cy.wrap(hdqDatabase.instances).each((instance) => {...```